frontend/src/api/models/survey.ts

```diff
@@ -1,6 +1,8 @@
+export type QuestionTypeName = 'Single Choice' | 'Multiple Choice';
 export interface QuestionType {
   id: number;
-  name: string;
+  // ? Maybe would be better to use singleChoice/multipleChoice instead
```

Comment:
At some point in the future, we will have a `free_text` question type, maybe others will come also, not sure if this info clarifies anything.

Reply:
@edimossilva do you agree that if we follow the proposal of the comment, we would go back to the point of turning the types into an Enum?

Reply:
@andersonmalheiro , yes it makes sense to use enum instead of objects it is the default approach for this situation. The reason why I used objects is because it is faster to develop the back office UI. At the same time I think using enum or objects have similar impact on client side

@andrecrimb, this discussion is not related to your PR